### PR TITLE
Switch to Jenkins

### PR DESCRIPTION
**Pull request by jkassemi**
Jenkins instead of Travis for builds, uses the vetsgov Jenkins cluster and `Jenkinsfile`. Jenkins interface available via http://jenkins.vetsgov-internal/

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -73,6 +73,8 @@ node('vets-website-linting') {
     try {
       dockerImage.inside(args) {
         sh "cd /application && npm --no-color run test:coverage"
+        sh "cd /application && npm install -g codeclimate-test-reporter"
+        sh "cd /application && CODECLIMATE_REPO_TOKEN=fe4a84c212da79d7bb849d877649138a9ff0dbbef98e7a84881c97e1659a2e24 codeclimate-test-reporter < ./coverage/lcov.info"
```
Comment (jkassemi) [inline Jenkinsfile]: Travis suggests that this token be used publicly, though Code Climate suggests that it be stored as a secret, and that public use could allow others to send coverage reports to our endpoint.

I'm moving this over as is from the Travis implementation, but this should probably be regenerated and transferred in another task. /cc @crwallace

**Comment by jkassemi** [comment]
Travis removed as a requirement from the branch configuration. Once this is in place I'll uncheck the box next to its integration in the repo settings.

**Comment by jkassemi** [comment]
@crwallace - any chance we could get this in place today before the deployment?

**Review by knkski — APPROVED** [review]
`git grep -i travis` shows me a couple of files that still contain references to Travis. Do we want to update these as well?

```
.coveralls.yml
content/pages/playbook/design/code.html
content/pages/playbook/downloads/VHA_Style_Guide_508.pdf
docs/HowToDeploy.md
script/travis-build.sh
```

**Comment by jkassemi** [comment]
@knkski - that'd be great, can you grab those references? The colophon also references a few technologies we no longer use, so may need a more complete overhaul.

**Pull request merged**